### PR TITLE
[ANN] Add header-check.0.1.0, a tool to check all headers and manage them

### PR DESCRIPTION
**Pull request by lefessan**

### CODE REVIEW

File: packages/header-check/header-check.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+license: "LGPL-2.1-only WITH OCaml-LGPL-linking-exception"
+synopsis: "A tool to check and update source headers, using checksums"
+description: """\
+The header-check tool scans a source project, lists all existing headers and
+can be used to update them. It uses checksums to create identifiers for 
+headers and works with OCaml, C/C++ and Shell kinds of comments.
+"""
+authors: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+maintainer: ["Fabrice Le Fessant <fabrice.le_fessant@ocamlpro.com>"]
+homepage: "https://ocamlpro.github.io/header-check"
```
Comment (raphael-proust) [inline packages/header-check/header-check.0.1.0/opam:11]: I have a 404 on this link. Is this expected for now? Is there a different URL to add?

Same remark on the doc link below.

This is not necessarily blocking, but if you want to fix it, you might as well do it before the pull.

Comment (lefessan) [reply packages/header-check/header-check.0.1.0/opam:11]: Thanks for noticing, I decided to push a website there, so it should be ok for now.

File: packages/header-check/header-check.0.1.0/opam [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "dune" {>= "2.7.0"}
+  "ocplib_stuff" {>= "0.1"}
+  "ez_file" {>= "0.3"}
+  "base-unix" {>= "base"}
+  "ppx_inline_test" {with-test}
+  "ppx_expect" {with-test}
+  "odoc" {with-doc}
+  "ocamlformat" {with-test}
```
[the remark this answers is not in the document]
Comment (lefessan) [reply packages/header-check/header-check.0.1.0/opam:44]: Actually, I would prefer to wait and see, and add versions only when it becomes necessary.

File: packages/header-check/header-check.0.1.0/opam (hunk 1 above)
Comment (mseri) [reply packages/header-check/header-check.0.1.0/opam:11]: It's working now

**Comment by kit-ty-kate** [comment]
Thanks

**Pull request merged**